ceilometer-central/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM FROM satellite.ugent.be:5000/hpc-ugent-testing-tier1_osp16-osp16_containers-ceilometer-central:16.1
```

Comment:
have you tried with the latest version instead of `16.1` tag?

Reply:
tags changed to 16.2 by 3328acd0193535886c9cda8ac8d6830b7e9bb503